app/controllers/surveys_controller.rb

```diff
@@ -70,37 +70,38 @@ def connect_call
       @next_question = @survey.survey_questions.find_by(id: @next_question_id)
 
       # Respond with some TwiML to kick-off the survey
-      response = Twilio::TwiML::Response.new do |r|
-
-        # If it's an input question use Gather
-        if @next_question[:recording] == false
-          r.Gather :numDigits => '1', :action => '/api/get_answer', :method => 'get' do |g|
-            g.Say @next_question[:body], voice: 'alice', language: 'en-GB'
-            g.Say '0 is the lowest. 9 is the highest.', voice: 'man', language: 'en-US'
-          end
-
-        # If it's a free-form question record the user
-        else
-          r.Say @next_question[:body], voice: 'alice', language: 'en-GB'
-          r.Say "Please record your response after the beep.", voice: 'alice', language: 'en-GB'
-          r.Record :timeout => '5', :method => 'get', :action => '/api/get_answer', :finishOnKey => '#', :maxLength => '90'
-        end
-
+      response = Twilio::TwiML::VoiceResponse.new
+
+      pp @next_question
+      # If it's an input question use Gather
+      if @next_question[:recording] == false
+        g = Twilio::TwiML::Gather.new :numDigits => '1', :action => "/api/get_answer", :method => 'get'
```

Comment:
Function call like this should use parentheses (from: https://github.com/bbatsov/ruby-style-guide)